pkg/xen-tools/initrd/eve-enter-container

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Do we really need this additional step here? I would integrate it with the agetty login, so as soon as user login into the prime-cons it would execute the command needed to get into the container.... is there any particular reason to allow the user login into the initrd environment?

Reply:
I would like to keep prime-cons as a mechanism to debug a hosting Vm.